src/mame/heathkit/tlb.cpp

```diff
@@ -1581,11 +1574,7 @@ void heath_imaginator_tlb_device::mem_map(address_map &map)
 
 void heath_imaginator_tlb_device::tap_6000h()
 {
-	if (m_mem_map != 0)
-	{
-		m_mem_map = 0;
-		m_mem_view.select(m_mem_map);
-	}
+	m_mem_bank->set_entry(0);
 }
```

Comment:
As far as I can tell, the only way for it to flip back to entry 1 is on reset, so you could make it uninstall the tap after the first time it’s hit to make it a bit more efficient.

Reply:
@cuavas How do you uninstall a tap? I looked for a method that would do it and didn't see anything.

Reply:
With `.remove()`: 706cc60b159b337545748c0abe0545b447037c0a

You can find a fair few uses of read/write taps around the source tree if you grep for them.